src/debug/daccess/dacdbiimpl.h

```diff
@@ -349,6 +349,26 @@ class DacDbiInterfaceImpl :
     // Returns true if the argument is a runtime callable wrapper
     BOOL IsRcw(VMPTR_Object vmObject);
 
+    HRESULT GetMethodDescPtrFromIpEx(
```

Comment:
I don't see any place where mscordbi calls this function. You could make this private and exclude from the interface.